beacon-chain/p2p/discovery_test.go

```diff
@@ -340,3 +343,163 @@ func addPeer(t *testing.T, p *peers.Status, state peerdata.PeerConnectionState)
 	}))
 	return id
 }
+
+func TestRefreshENR_ForkBoundaries(t *testing.T) {
+	params.SetupTestConfigCleanup(t)
+	// Clean up caches after usage.
+	defer cache.SubnetIDs.EmptyAllCaches()
+	defer cache.SubnetIDs.EmptyAllCaches()
```

Comment:
Is it necessary to do this twice?

Reply:
ah good catch , not it was an oversight will fix it.